### DISCUSSION TITLE
[Script request] Manyfold

### DISCUSSION THREAD

**Discussion by DeltaAngle** [community-scripts/ProxmoxVE#425] [Request script]
### Application Name

Manyfold

### Website

https://github.com/manyfold3d/manyfold

### Description

Manyfold is a web application for managing and organizing 3D models, with a focus on models for 3D printing. Adding Manyfold to helper-scripts would make it easier for users to host this application on Proxmox and manage their models. This would be a nice accompaniment to Octoprint.

### Due Diligence

- [X] I have searched existing [scripts](https://community-scripts.github.io/Proxmox/scripts) and found no duplicates.
- [X] I have searched existing [discussions](https://github.com/community-scripts/ProxmoxVE/discussions?discussions_q=) and found no duplicate requests.
- [X] This is not a game-related request.

**Comment by TopsideWings** [comment]
There's a Manyfold script now, but not quite working for me..

**Reply by IvanAS94** [reply]
same, the install just fail

**Reply by KoekieWaus** [reply]
He does install it here. But then I have to choose folders, and it keeps freezing there.
